packages/block-library/src/image/deprecated.js

```diff
@@ -941,4 +941,214 @@ const v7 = {
 	},
 };
 
-export default [ v7, v6, v5, v4, v3, v2, v1 ];
+const v8 = {
+	attributes: {
+		align: {
+			type: 'string',
+		},
+		behaviors: {
+			type: 'object',
+		},
```

Comment:
@ajlende I've added the block deprecation, but the `migrate()` function would not run without the `behaviors` attribute defined here, even though we never had or needed it in the `attributes` definition to make the `behaviors` syntax work.

Is this how the block deprecation should be done?

Reply:
> Is this how the block deprecation should be done?

Yes.

The block used to have this which added the attribute.

```json
{
	"supports": {
		"behaviors": {
			"lightbox": true
		}
	}
}
```

Since your plan is to entirely remove the block supports, adding this attribute is the way to do the deprecation. You'll want to add a dev note explaining this for other folks who used the lightbox block support. Maybe even link it in the deprecation notice when the block support runs.

Reply:
> Also be sure to update the other block deprecations that had the support to include the attribute instead.

@michalczaplinski Just double checking: We only need to do this for the image in its `deprecated.js`, right? You didn't add these supports to any other blocks?

Reply:
Nope, no block other than Image had the `supports.behaviors` added.